tests/baselines/reference/api/typescript.d.ts

```diff
@@ -2254,6 +2254,7 @@ declare namespace ts {
         JsxAttributes = 4096,
         MarkerType = 8192,
         JSLiteral = 16384,
+        FreshLiteral = 32768,
```

Comment:
So this is now intended for public use or did you just forget to make it internal?